xml/deployment_hwrecommend.xml

```diff
@@ -375,11 +375,12 @@
   <title>Network Recommendations</title>
 
   <para>
-   The network environment where you intend to run &ceph; should ideally be a
-   bonded set of at least two network interfaces that is logically split into a
-   public part and a trusted internal part using VLANs. The bonding mode is
-   recommended to be 802.3ad if possible to provide maximum bandwidth and
-   resiliency.
+   For the &ceph; cluster network environment, we recommend a bonded set of at
+   least two separate 25 GbE network interfaces. The networks interfaces should
```

Comment:
two bonded 10 GbE or one 25 GbE AFAIK

Reply:
@ajaeger so.....how far did we get with the NIC specifications? any last suggestion?

Reply:
Might take a bit longer to get this done...

Reply:
ok, when should i ping you again, in a week?

Reply:
@tbazant - Kai's response below answers this query.